runtime/jcl/uma/jfr_exports.xml

```diff
@@ -21,4 +21,69 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 -->
 <exports group="jfr">
 	<export name="Java_com_ibm_oti_vm_VM_triggerExecutionSample" />
+	<export name="Java_jdk_jfr_internal_JVM_registerNatives" />
```

Comment:
youll also need a objects.xml for UMA build system

Reply:
then youll need to update runtime/jcl/module.xml to add both the objects and the exports file

Reply:
Sorry for the late reply, I've done the changes